python/NLP/subscriber.py

```diff
@@ -0,0 +1,64 @@
+import json
+import asyncio
+from datetime import datetime
+
+from pyensign.ensign import Ensign
+from pyensign.events import Event
+from pyensign.api.v1beta1.ensign_pb2 import Nack   
+
+
+
+TOPIC = "documents"
+
+# An asyncio Event object manages an internal flag that can be set to true with the
+# set() method and reset to false with the clear() method. The wait() method blocks
+# until the flag is set to true. The flag is set to false initially.
+# That means we can use it to communicate between the publisher (which should create a
+# "wait" signal) and the subscriber (which will acknowledge the event with `set`)
+RECEIVED = asyncio.Event()
+
+
+async def print_single_event(event):
+    
+    msg = json.loads(event.data)
+    
+    print(msg)
+
+ 
+
+async def handle_event(event):
+    """
+    Decode and ack the event.
+    """
+    try:
+        data = json.loads(event.data)
```

Comment:
@Aatmaj1 this is the line that is the problem (see slack discussion in #baleen channel for full details)

The `event` is not something the `json` library can read (yet). We need a Python library that will unpack the event correctly. Can you look for one that knows how to marshal/unmarshal `msgpack` format?